File: packages/insomnia-httpsnippet/test/fixtures/output/python/python3/application-json.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "{\"number\":1,\"string\":\"f\\\"oo\",\"arr\":[1,2,3],\"nested\":{\"a\":\"b\"},\"arr_mix\":[1,\"a\",{\"arr_mix_nested\":{}}],\"boolean\":false}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: packages/insomnia-httpsnippet/test/fixtures/output/node/native/application-json.js
var http = require('http');

var options = {
  method: 'POST',
  hostname: 'mockbin.com',
  port: null,
  path: '/har',
  headers: {
    'content-type': 'application/json',
  },
};

var req = http.request(options, function(res) {
  var chunks = [];

  res.on('data', function(chunk) {
    chunks.push(chunk);
  });

  res.on('end', function() {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write(
  JSON.stringify({
    number: 1,
    string: 'f"oo',
    arr: [1, 2, 3],
    nested: { a: 'b' },
    arr_mix: [1, 'a', { arr_mix_nested: {} }],
    boolean: false,
  }),
);
req.end();
